index.bs

```diff
@@ -5313,22 +5313,32 @@ The following types are known as <dfn id="dfn-integer-type" export>integer types
 {{unsigned short}},
 {{long}},
 {{unsigned long}},
-{{long long}} and
-{{unsigned long long}}.
+{{long long}},
+{{unsigned long long}},
+any [=nullable types=] whose [=nullable types/inner type=] is an [=integer type=],
+and any [=annotated types=] whose [=annotated types/inner type=] is an [=integer type=].
 
 The following types are known as <dfn id="dfn-numeric-type" export>numeric types</dfn>:
 the [=integer types=],
 {{float}},
 {{unrestricted float}},
-{{double}} and
-{{unrestricted double}}.
+{{double}},
```

Comment:
So... With this change there are a few things that are a bit odd:

1) https://heycam.github.io/webidl/#es-overloads step 12.12 no longer needs to worry about "a nullable numeric type" or "an annotated type whose inner type is one of the above types", since those will be included under "numeric type", no?  Same thing for step 12.14.  And 12.13 given the "string type" changes below.

2) There are various other places in the spec that mention "numeric types" but they all already handle unwrapping nullable and annotated types.  Maybe this is OK; it's just a little confusing to consider the table in the distinguishability section, for example, when "numeric types" could in theory include `long?` but in practice can't because we unwrapped that already...